js/accounts/auth.js

```
@@ -50,6 +52,40 @@ export class AuthManager {
+            // Poll for credential from the system browser callback
+            for (let i = 0; i < 120; i++) {
+                await new Promise((r) => setTimeout(r, 1500));
+                try {
+                    const res = await fetch(`/api/auth/desktop-poll?sessionId=${encodeURIComponent(sessionId)}`);
+                    const data = await res.json();
+                    if (data.ok) {
+                        const credential = GoogleAuthProvider.credential(data.idToken, data.accessToken);
+                        const result = await signInWithCredential(auth, credential);
+                        this.user = result.user;
+                        this.updateUI(result.user);
+                        this.authListeners.forEach((listener) => listener(result.user));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)